internal/util/errors.go

```diff
@@ -0,0 +1,12 @@
+//
+// Copyright Strimzi authors.
+// License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+//
+
+// Package util contains some utility functions
```

Comment:
Is this really a _utility function_? I wouldn't normally complain ... but the comment makes it weird :-o

Reply:
I removed this file in my latest commit :-)